### PR TITLE
jira -> issue type index

### PR DESCRIPTION
**Pull request by lwsanty**
close [[DEV-1447]]

Signed-off-by: lwsanty <lwsanty@gmail.com>

[DEV-1447]: https://athenianco.atlassian.net/browse/DEV-1447

### CODE REVIEW

File: sql/migrations/098_jira_issue_type_index.up.sql [hunk 1]
```diff
@@ -0,0 +1 @@
+CREATE INDEX issue_type_name ON jira.issue_type(acc_id, project_id, name);
```
Comment (vmarkovtsev) [inline sql/migrations/098_jira_issue_type_index.up.sql:1]: AFAIK Denys wants
```
BEGIN;
CREATE/DROP INDEX;
COMMIT;
```

**Review by vmarkovtsev — CHANGES_REQUESTED** [review]

File: sql/migrations/098_jira_issue_type_index.up.sql (hunk 1 above)
Comment (vmarkovtsev) [reply sql/migrations/098_jira_issue_type_index.up.sql:1]: BTW `CREATE INDEX issue_type_name ON jira.issue_type(acc_id, project_id, name) INCLUDE(icon_url);` will be the best :+1:

Comment (lwsanty) [reply sql/migrations/098_jira_issue_type_index.up.sql:1]: afair this convention is not related to one-liners

Comment (lwsanty) [reply sql/migrations/098_jira_issue_type_index.up.sql:1]: cc @dennwc

**Review by vmarkovtsev — APPROVED** [review]

File: sql/migrations/098_jira_issue_type_index.up.sql [hunk 2]
```diff
@@ -0,0 +1 @@
+CREATE INDEX issue_type_name ON jira.issue_type(acc_id, project_id, name) INCLUDE(icon_url);
```
Comment (dennwc) [inline sql/migrations/098_jira_issue_type_index.up.sql:1]: What about existing `issue_type` index?

**Review by dennwc — CHANGES_REQUESTED** [review]

File: sql/migrations/098_jira_issue_type_index.up.sql (hunk 2 above)
Comment (dennwc) [reply sql/migrations/098_jira_issue_type_index.up.sql:1]: Ah, sorry this index in on `issue`. Anyway, can you please change `issue_type` index on `issue` to include `acc_id` as well (as the first column)?

Comment (lwsanty) [reply sql/migrations/098_jira_issue_type_index.up.sql:5]: Done.

**Review by dennwc — APPROVED** [review]

**Pull request merged**